examples/index.html

```diff
@@ -564,7 +589,7 @@
 
 		}
 
-		</script>
+		</script><div class="marquee"><p>refresh_fix_on_some_blacklisted_android_chipsets</p></div></article>
```

Comment:
Hold on a second. Blacklisted... does it mean that you've disabled the blacklisting? If so, wouldn't it be better to just support Firefox? I suspect very few people disable blacklisting.